simply/market.py

```diff
@@ -48,6 +55,28 @@ def print(self):
         print(self.get_bids())
         print(self.get_asks())
 
+    def filter_market_maker(self, order, ignore_index=True, index=None):
```

Comment:
Filter function should not implicitly filter large bids/asks. Instead, the thresholds MARKET_MAKER_THRESHOLD and LARGE_ORDER_THRESHOLD should be used, i.e.
`if   MARKET_MAKER_THRESHOLD > order.energy > LARGE_ORDER_THRESHOLD
  ....
elif MARKET_MAKER_THRESHOLD < order.energy
  ....
`
This would make some conditions at the function call unnecessary.

Right now, if I would filter a small order by mistake, it put the order inside the "large" dataframes.
At the same time this filter function could also take care of mutating self.orders, so most conditions around the  filter call could vanish